### DISCUSSION TITLE
Have Tilt component overlay on another component?

### DISCUSSION THREAD

**Discussion by matthewpsbilo** [mkosir/react-parallax-tilt#37] [Q&A] [answered]
Hello! I would just like to ask if it's possible to override the style of the Tilt component so I can put the Tilt component on top of a static component. 

Thank you very much!

**Comment by mkosir** [comment, marked as answer]
Hey @mpsb 👋 
not sure if I understand correctly, but you can style `Tilt` component with `className` and of course apply any kind of styling to "inner" component that you are wrapping it, like in [this](https://mkosir.github.io/react-parallax-tilt/?path=/story/react-parallax-tilt--parallax-effect) example.
